src/site/content/en/learn-web-vitals/crux-and-rum-differences/index.md

```diff
@@ -0,0 +1,206 @@
+
+RUM data will allow segmenting traffic similarly, but often shows consolidated data by default. RUM may only allow segmenting by device type (for example, mobile) or browser (for example, Chrome) easily, but not both to only see mobile Chrome traffic. When comparing to CrUX data, ensure you are comparing like-for-like by filtering by device type _and_ the Chrome browser.
+
+#### Sampling
+
+RUM solutions typically allow adjusting the sampling rate of opted-in visitors where data is collected. This can be used to reduce the volume of data needed to be analyzed, and to reduce costs of commercial RUM services. If that sample size is too small and not representative of the wider population, then the resulting metrics will also be similarly skewed. Discuss with your RUM provider the appropriate sampling size for your site.
+
+### Aggregation of data
+
+Field data by its very nature will include many, many data points of the same metrics compared to lab data, which will give a single value. If this data is aggregated differently for reporting, it can lead to another reason for differences between CrUX and RUM.
+
+#### Timespan
+
+CrUX data is based on a 28 day sliding window of traffic, and it is not possible to change this timeframe—though the BigQuery data is stored for each month, allowing you to see previous months.
+
+RUM data typically allows much greater granularity to allow seeing the impact of changes much sooner. When choosing smaller periods, though, RUM data can be unduly impacted by fluctuations in website traffic and visitors. When comparing RUM data to CrUX data, always ensure you are looking at performance over 28 days. Once you are satisfied the data is similar, you can look at other time frames to drill into the RUM data.
+
+#### Aggregation of statistics
+
+CrUX metrics are measured at the 75th percentile—that is, looking at the value that 75% of page views achieved. There will be extremes in field data and removing the worst 25% experiences, it is intended to give a value that the majority of visitors can reasonably be expected to achieve.
+
+RUM products often give a wider number of options of how to aggregate the metrics, including 75th percentile, median and other percentiles. If comparing RUM values to CrUX data, it is necessary to ensure you are looking at 75th percentile data to compare like-for-like.
+
+The histogram data in CrUX includes all available data—not just 75th percentile—and shows the number of page views in each rating, but the aggregate score will be based on 75th percentile. This CrUX data is surfaced in tools like [PageSpeed Insights](https://pagespeed.web.dev/):
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/oRCstCFf6JyiVlNuphuX.png", alt="PageSpeed Insights screenshot showing histograms of LCP rating page loads", width="632", height="428" %}
```

Comment:
```suggestion
<figure>
  {% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/oRCstCFf6JyiVlNuphuX.png", alt="PageSpeed Insights screenshot showing histograms of LCP rating page loads", width="632", height="428" %}
</figure>
```

Reply:
Optional suggestion, but since this is a figure, it probably should be in a `<figure>` element.